forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -269,17 +269,21 @@ class ForageSDK : ForageSDKInterface {
     }
 
     /**
-     * Capture a customer's PIN for an EBT payment and defer the capture of the payment to the server
+     * Submits a customer's PIN via a [ForagePINEditText][com.joinforage.forage.android.ui.ForagePINEditText] Element and defers payment capture to the server.
+     *
+     * @param DeferPaymentCaptureParams A model that passes a [`foragePinEditText`][com.joinforage.forage.android.ui.ForagePINEditText] instance and a `paymentRef`, returned by the [Create a Payment](https://docs.joinforage.app/reference/create-a-payment) endpoint, as the [DeferPaymentCaptureParams].
      *
-     * @param params The parameters required for pin capture, including
-     * reference to a ForagePINEditText and a Payment ref
+     * @return A [ForageApiResponse] object. 
+     * 
+     * On success, the object returns `Nothing`. 
+     * 
+     * On failure, the response includes a list of [ForageError][com.joinforage.forage.android.network.model.ForageError] objects that you can unpack to troubleshoot the issue. 
+     * @throws [ForageConfigNotSetException] If the [ForageConfig] is not set for the provided `foragePinEditText`. 
      *
-     * @return A ForageAPIResponse indicating the success or failure of the
-     * PIN capture. On success, returns `Nothing`.
-     * On failure, provides a detailed error response.
+     * @see 
+     * [Defer EBT payment capture to the server](https://docs.joinforage.app/docs/capture-ebt-payments-server-side) for the related step-by-step guide.
      *
-     * @throws ForageConfigNotSetException If the passed ForagePINEditText instance
-     * hasn't had its ForageConfig set via .setForageConfig().
+     * [Capture an EBT Payment](https://docs.joinforage.app/reference/capture-a-payment) for the related endpoint.
```

Comment:
Maybe worth clarifying that this endpoint should be called after `deferPaymentCapture`